Mathlib/AlgebraicGeometry/Pullbacks.lean

```diff
@@ -620,4 +620,43 @@ instance Scheme.pullback_map_isOpenImmersion {X Y S X' Y' S' : Scheme}
   rw [pullback_map_eq_pullbackFstFstIso_inv]
   infer_instance
 
+section Spec
+
+variable (R S T : Type u) [CommRing R] [CommRing S] [CommRing T] [Algebra R S] [Algebra R T]
+
+open TensorProduct Algebra.TensorProduct CommRingCat RingHomClass
+
+/-- The isomorphism between the fiber product of two affine schemes `Spec B` and `Spec C`
+over an affine scheme `Spec A` and the `Spec` of the tensor product `B ⊗[A] C`.-/
```

Comment:
```suggestion
/-- The isomorphism between the fiber product of two schemes `Spec S` and `Spec T`
over a scheme `Spec R` and the `Spec` of the tensor product `S ⊗[R] T`.-/
```

Reply:
(I suggest removing the "affine"s because we are working in the category of schemes, and we are saying explicitly that the schemes are `Spec` of something so it is clear that they are affine.)